tests/conformance/test_conformance.py

```diff
@@ -217,8 +225,44 @@ def client_list_hmac_keys(client, _preconditions, **_):
         pass
 
 
-def client_get_service_account_email(client, _preconditions, **_):
-    client.get_service_account_email()
+def client_get_hmac_key_metadata(client, _preconditions, **resources):
+    access_id = resources.get("hmac_key").access_id
+    client.get_hmac_key_metadata(access_id=access_id)
+
+
+def hmac_key_exists(client, _preconditions, **resources):
+    access_id = resources.get("hmac_key").access_id
+    hmac_key = HMACKeyMetadata(client, access_id=access_id)
+    hmac_key.exists()
+
+
+def hmac_key_reload(client, _preconditions, **resources):
+    access_id = resources.get("hmac_key").access_id
+    hmac_key = HMACKeyMetadata(client, access_id=access_id)
+    hmac_key.reload()
```

Comment:
What is the use case for this? Don't really understand why there are multiple get methods. (but that's not a problem for this PR!)

Reply:
There are 3 entry points in the library that call `storage.hmacKey.get`, and we want to make sure all of them are retried correctly. `hmac_key.reload()` is often used after instantiating a `HMACKeyMetadata` and before making updates.